mandatory/2-logic-error.js

```diff
@@ -1,16 +1,21 @@
+/** @format */
+
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return wordtrim();
+	return word.trim();
 }
 
 function getStringLength(word) {
-  return "word".length();
+
+	var noSpaceString = word.trim();
+	return noSpaceString.length;
```

Comment:
I like the extra work you've done on this -  word.length would have been enough to pass the test, but adding trim is a good idea. When you come to writing tests yourself, think about how you could write a test that would cover this extra functionality.